### PR TITLE
replace geth import for patched websocket client

### PR DESCRIPTION
**Pull request by jmank88**
https://app.clubhouse.io/chainlinklabs/story/16033/chainlink-doesn-t-reconnect-when-the-websocket-connection-hangs

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -85,3 +85,5 @@ require (
 // If that happens to get released in a 3.X.X version, we can add a constraint to our go.mod
 // for it. If its in 4.X.X, then we need all our transitive deps to upgrade to it.
 replace github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.1+incompatible
+
+replace github.com/ethereum/go-ethereum => github.com/smartcontractkit/go-ethereum v1.9.26-0.20210909013251-729bcfdc948c
```
Comment (jmank88) [inline go.mod:89]: go mod reformatted this from the full commit hash [729bcfdc948c54702783bd7895a0078081388fcd](https://github.com/smartcontractkit/go-ethereum/commit/729bcfdc948c54702783bd7895a0078081388fcd)

File: go.mod [hunk 2]
```diff
@@ -85,3 +85,6 @@ require (
 // If that happens to get released in a 3.X.X version, we can add a constraint to our go.mod
 // for it. If its in 4.X.X, then we need all our transitive deps to upgrade to it.
 replace github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.1+incompatible
+
+// severed-websocket-1.10.8
+replace github.com/ethereum/go-ethereum => github.com/smartcontractkit/go-ethereum v1.10.9-0.20210909121856-4edaf99ef949
```
Comment (jmank88) [inline go.mod:90]: https://github.com/smartcontractkit/go-ethereum/commit/4edaf99ef94987ec7b02873288fa12a249b7f617

Comment (jmank88) [reply go.mod:90]: Until geth includes https://github.com/ethereum/go-ethereum/pull/23556

File: go.mod [hunk 3]
```diff
@@ -85,3 +85,6 @@ require (
 // If that happens to get released in a 3.X.X version, we can add a constraint to our go.mod
 // for it. If its in 4.X.X, then we need all our transitive deps to upgrade to it.
 replace github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.1+incompatible
+
+// severed-websocket-1.10.8
+replace github.com/ethereum/go-ethereum => github.com/smartcontractkit/go-ethereum v1.10.9-0.20210909134823-a177d470d620
```
Comment (jmank88) [inline go.mod:90]: https://github.com/smartcontractkit/go-ethereum/commit/a177d470d62012c5adc4e74bcfc946a6977dc040
Until geth includes https://github.com/ethereum/go-ethereum/pull/23556

**Review by j16r — APPROVED** [review]

**Pull request merged**